### PR TITLE
fixing various issues noted on github

### PR DESCRIPTION
**Pull request by planes**
This PR addresses #391, #390, #387, and #386

### CODE REVIEW

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/induced_drag_aircraft.py [hunk 1]
```diff
@@ -133,6 +133,7 @@ def induced_drag_aircraft(state,settings,geometry):
         for wing in wings:
             if wing.areas.reference>S:
                 AR = wing.aspect_ratio
+                S  = wing.areas.reference
```
Comment (mclarke2) [inline trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/induced_drag_aircraft.py:136]: This doesn't look like it adds anything. If it is not used, remove it

**Review by mclarke2 — COMMENTED** [review]
One thing needs to be addressed in the induced_drag_aircraft.py

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/induced_drag_aircraft.py (hunk 1 above)
Comment (planes) [reply trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/induced_drag_aircraft.py:136]: No this is a necessary line that was missing. This looks for the largest wing and decides it's the main wing to use as the AR. The S helps tracks the wings aspect ratio.

See #387

Comment (mclarke2) [reply trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/induced_drag_aircraft.py:136]: Ahhh ok. Thanks for clarifying.

**Review by mclarke2 — APPROVED** [review]

**Pull request merged**